docroot/themes/humsci/humsci_basic/src/scss/_base.scss

```diff
@@ -0,0 +1,18 @@
+html {
+  font-size: 1em;
+
+  @include colorful {
+    color: black;
+  }
+  @include airy {
+    color: red;
+  }
+  @include traditional {
+    color: yellow;
+  }
+
+  // TODO: Create mixin to support this
+  //@include theme(colorful, airy) {
+  //  background-color: gray;
+  //}
```

Comment:
Example usage of a mixin that takes multiple themes as parameters. We still need to create the mixin that would do this.